code/modules/mob/living/carbon/life.dm

```diff
@@ -547,6 +547,10 @@ GLOBAL_LIST_INIT(ballmer_windows_me_msg, list("Yo man, what if, we like, uh, put
 
 //used in human and monkey handle_environment()
 /mob/living/carbon/proc/natural_bodytemperature_stabilization()
+	//NSV13 - coldblooded beings do not naturally stabilize.
+	if(HAS_TRAIT(src, TRAIT_COLDBLOODED))
+		return 0
+	//NSV13 end.
```

Comment:
```suggestion
	//NSV13 - coldblooded beings do not naturally stabilize.
	if(HAS_TRAIT(src, TRAIT_COLDBLOODED))
		return 0
	//NSV13 end.
```
You could also simulate this by just lowering the metabolism efficiency a lot. Cold-blooded animals still generate *some* metabolic heat.

Reply:
Ectothermism is the inability to *control* body-temperature. Metabolism is essentially a constant factor for heat (albeit potentially slightly higher or lower depending on activity).

Emulating the effect of metabolism on body heat would be a constant heat gain over time for (almost) all species, which would be much more punishing than this is, especially on NSV where having to suit up is common.

Reply:
Active thermogenesis via means such as muscle activity for the sake of exotherm energy usage could be considered I suppose, while ignoring the default metabolism inherent to most beings for sake of not causing things to melt in any insulated equipment.

This would delta exclusively upwards, be slow, and require additional nutrition to maintain (while below a certain temperature)

Reply:
My take on this sort of behavior implemented. See testing section for a demo.